.github/workflows/ci.yml

```diff
@@ -108,8 +112,6 @@ jobs:
       run: ./bootstrap.sh
     - name: Run v3 UI Tests (Debug)
       run: bundle exec fastlane test_ui device:"iPhone 12"
-    - name: Post Codecov report
-      run: bash <(curl -s https://codecov.io/bash) -t ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Why don't we post codecov for UI tests?

Reply:
Why don't we post it with a flag, too?

Reply:
We had the code coverage off when running the UI Tests with fastlane's scan because we were in the early stages of the Audits. But I guess we can turn it on now, so we can track the progress 🤔  I think it is a good idea 👍  WDYT?  cc @VojtaStavik

Reply:
This flag basically guarantees we don't mess up coverages, right? It's safe to enable in that case

Reply:
Yes exactly